tools/pd-ctl/tests/scheduler/scheduler_test.go

```diff
@@ -315,7 +321,7 @@ func (suite *schedulerTestSuite) checkScheduler(cluster *pdTests.TestCluster) {
 	echo = mustExec(re, cmd, []string{"-u", pdAddr, "scheduler", "remove", "evict-leader-scheduler-2"}, nil)
 	re.Contains(echo, "Success!")
 	echo = mustExec(re, cmd, []string{"-u", pdAddr, "scheduler", "remove", "evict-leader-scheduler-1"}, nil)
-	re.Contains(echo, "404")
+	re.Contains(echo, "Failed! scheduler evict-leader-scheduler not found")
```

Comment:
Do other schedulers need it? For example grant-leader-scheduler.

Reply:
yes, I added in `grant-leader-scheduler` and `evict-leader-scheduler`